src/Silex/Application.php

```diff
@@ -91,12 +98,32 @@ public function __construct(array $values = array())
             return new CallbackResolver($app);
         };
 
+        $this['argument_resolvers'] = function ($app) {
```

Comment:
As we won't bump the requirement of Symfony to 2.8 for Silex 2.0, the code should be compatible with both 2.7 and 2.8.